app/connector/rest-swagger/src/main/java/io/syndesis/connector/rest/swagger/SwaggerConnectorComponent.java

```diff
@@ -46,6 +47,8 @@
 
     private String accessToken;
 
+    private Date expiresAt;
```

Comment:
Perhaps `long` instead?

Reply:
Date and long are pretty much the same thing. I prefer Date since that's what it is.